readme.txt

```diff
@@ -4,7 +4,7 @@ Donate link: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=mrtor
 Tags: SEO, all in one seo, WordPress SEO, Google Search Console, XML Sitemap, image seo
 Requires at least: 4.4
 Tested up to: 5.0
-Stable tag: 2.10
```

Comment:
Why is this file being modified?

This should be left for the Version Change PR. It is also outdated.